src/client/Client.ts

```diff
@@ -31,6 +32,7 @@ class Spencer extends Client {
 	public prefix: string;
 	public owners: Array<string>;
 	public config: Config;
+	public vacefron: VACEFronJS;
```

Comment:
initiate the class here

Reply:
Do you mean like `public vacefron: VACEFronJS = new VACEFronJS()`?

Reply:
Yes

Reply:
Alright